mara_db/shell.py

```diff
@@ -83,6 +83,9 @@ def __(db: dbs.BigQueryDB, timezone: str = None, echo_queries: bool = None):
     echo_queries = None
     assert all(v is None for v in [timezone, echo_queries]), f"unimplemented parameter for BigQueryDB"
 
+    # One time activation of the service-account used is required
+    # todo: maybe as part of db initialization
+    # gcloud auth activate-service-account --key-file='path-to/service-account.json'
```

Comment:
Can we move this into the README or into the documentation of the init-method of class BigQueryDB?

Reply:
Yes, I was also thinking to move it in a more transparent place.

Thanks for the review @hz-lschick 
In addition, please note that this PR is still missing the **copy_to_stdout_command** generic implementation in a way that can be used without issues from the **copy_command** (i.e. from BQ to Postrgres)